src/app/api/services/dot-devices/dot-devices.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { Device } from '../../../shared/models/device/device.model';
+
+/**
+ * Provide util methods to get the Devices & dimensions.
+ * @export
+ * @class DotDevicesService
+ */
+@Injectable()
+export class DotDevicesService {
+    constructor() {}
+
+    /**
+     * Return available devices.
+     * @returns {Observable<Device[]>}
+     * @memberof DotDevicesService
+     */
+    get(): Observable<Device[]> {
+        return Observable.of([
+            { id: '0', label: 'Desktop', width: '100%', height: '100%' },
+            { id: '1', label: 'iPhone', width: '375px', height: '667px' },
+            { id: '2', label: 'iPad Pro', width: '1024px', height: '1366px' },
+            { id: '3', label: 'iPad 3, 4, Air, Air2', width: '768px', height: '1024px' }
+        ]);
```

Comment:
this have to be request from the BE in the future, so add A TODO here

Reply:
Rori is this hardcoded on the jsp in the current BE or is it from the DB?

Reply:
Hey guys I just check the code and the devices are content, you can pull the info using the rest service content:
```
https://demo.dotcms.com/api/content/render/false/query/+contentType:previewDevice
```